src/Mutation.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Softonic\GraphQL;
+
+use Softonic\GraphQL\Config\MutationConfigObject;
+use Softonic\GraphQL\Mutation\Item;
+use Softonic\GraphQL\Query\Item as QueryItem;
+
+class Mutation
+{
+    private $mutation;
+
+    private $config;
+
+    /**
+     * @param array<MutationConfigObject> $mutationConfig
+     */
+    public function __construct(array $mutationConfig, QueryItem $source)
+    {
+        $this->config = $mutationConfig['program'];
```

Comment:
What's the purpose of this item (`$mutationConfig['program']`)?

Reply:
Still something hardcoded that should be changed.